File: event/src/test/java/de/danoeh/antennapod/event/FeedUpdateRunningEventTest.java
package de.danoeh.antennapod.event;

import org.junit.Test;

import static org.junit.Assert.assertFalse;
import static org.junit.Assert.assertTrue;

public class FeedUpdateRunningEventTest {

    @Test
    public void trueIsStoredAndReadBack() {
        FeedUpdateRunningEvent event = new FeedUpdateRunningEvent(true);
        assertTrue(event.isFeedUpdateRunning);
    }

    @Test
    public void falseIsStoredAndReadBack() {
        FeedUpdateRunningEvent event = new FeedUpdateRunningEvent(false);
        assertFalse(event.isFeedUpdateRunning);
    }
}

File: event/src/test/java/de/danoeh/antennapod/event/FeedUpdateRunningEventTest.kt
package de.danoeh.antennapod.event

import org.junit.Assert.assertFalse
import org.junit.Assert.assertTrue
import org.junit.Test

class FeedUpdateRunningEventTest {

    @Test
    fun trueIsStoredAndReadBack() {
        val event = FeedUpdateRunningEvent(true)
        assertTrue(event.isFeedUpdateRunning)
    }

    @Test
    fun falseIsStoredAndReadBack() {
        val event = FeedUpdateRunningEvent(false)
        assertFalse(event.isFeedUpdateRunning)
    }
}
